velox/benchmarks/tpch/TpchBenchmark.cpp

```diff
@@ -155,21 +156,74 @@ int main(int argc, char** argv) {
     folly::runBenchmarks();
   } else {
     const auto queryPlan = queryBuilder->getQueryPlan(FLAGS_run_query_verbose);
-    const auto task = benchmark.run(queryPlan);
-    ensureTaskCompletion(task.get());
-    const auto stats = task->taskStats();
-    std::cout << fmt::format(
-                     "Execution time: {}",
-                     succinctMillis(
-                         stats.executionEndTimeMs - stats.executionStartTimeMs))
-              << std::endl;
-    std::cout << fmt::format(
-                     "Splits total: {}, finished: {}",
-                     stats.numTotalSplits,
-                     stats.numFinishedSplits)
-              << std::endl;
-    std::cout << printPlanWithStats(
-                     *queryPlan.plan, stats, FLAGS_include_custom_stats)
-              << std::endl;
+    std::vector<double> executionTime;
+
+    for (int iter = 0; iter <= FLAGS_num_runs; iter++) {
+      const auto task = benchmark.run(queryPlan);
+      ensureTaskCompletion(task.get());
+      const auto stats = task->taskStats();
+
+      if (iter == 0) {
+        executionTime.push_back(
+            stats.executionEndTimeMs - stats.executionStartTimeMs);
+
```

Comment:
These 3 lines are common across both sides of the condition. So they can be moved outside it.